ecl/eclcc/eclcc.cpp

```diff
@@ -374,11 +374,13 @@ class EclCC : public CInterfaceOf<ICodegenContextCallback>
     bool logTimings;
     bool optArchive;
     bool optCheckEclVersion;
+    bool optDebugMemLeak = false;
     bool optEvaluateResult;
     bool optGenerateMeta;
     bool optGenerateDepend;
     bool optIncludeMeta;
     bool optKeywords;
+    bool optReleaseMem = false;
```

Comment:
Do we want to preserve previous behaviour by making this true for #ifdef _DEBUG?